### PR TITLE
(DOCSP-16690) Backfills docs content for networking commands

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

Backfills docs content for networking commands

_Jira ticket:_
https://jira.mongodb.org/browse/DOCSP-16690
https://jira.mongodb.org/browse/DOCSP-16691
https://jira.mongodb.org/browse/DOCSP-16693
https://jira.mongodb.org/browse/DOCSP-16694
https://jira.mongodb.org/browse/DOCSP-16695
https://jira.mongodb.org/browse/DOCSP-16697
https://jira.mongodb.org/browse/DOCSP-16698
https://jira.mongodb.org/browse/DOCSP-16699

## Checklist
- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/networking/peering/create/azure.go [hunk 1]
```diff
@@ -123,10 +123,14 @@ func AzureBuilder() *cobra.Command {
...
 	cmd := &cobra.Command{
 		Use:   "azure",
-		Short: "Create a connection with Azure.",
-		Example: fmt.Sprintf(`  # The following command creates a peering connection between the Atlas VPC and your Azure VNet for a project using the default profile:
-  %s networking peering create azure --atlasCidrBlock 192.168.0.0/21 --directoryId 56657fdb-ca45-40dc-fr56-77fd8b6d2b37 --subscriptionId 345654f3-77cf-4084-9e06-8943a079ed75 
-  --resourceGroup atlascli-test --region US_EAST_2 --vnet atlascli-test`, cli.ExampleAtlasEntryPoint()),
+		Short: "Create a network peering connection between the Atlas VPC and your Azure VNet.",
+		Long: `Before you create an Azure network peering connection, complete the prerequisites listed here: https://www.mongodb.com/docs/atlas/reference/api/vpc-create-peering-connection/#prerequisites.
+		
+		The network peering create command checks if a VNet exists in the region you specify for your Atlas project. If one exists, this command creates the peering connection between that VNet and your VNet. If an Atlas VNet does not exist, this command creates one and creates a connection between it and your VNet.
```
Comment (kyuan-mongodb) [inline internal/cli/atlas/networking/peering/create/azure.go:129]: Question: On the Atlas side, it's supposed to be VPC right?

```suggestion
		The network peering create command checks if a VNet exists in the region you specify for your Atlas project. If one exists, this command creates the peering connection between that VNet and your VPC. If an Atlas VPC does not exist, this command creates one and creates a connection between it and your VNet.
```

File: internal/usage/usage.go [hunk 2]
```diff
@@ -262,18 +262,18 @@ dbName and collection are required only for built-in roles.`
...
+	AtlasCIDRBlock                            = "CIDR block that Atlas uses for all network peering connections created in the project. Required only if you do not already have an Atlas VPC. To learn more, see the Atlas UI tab at https://www.mongodb.com/docs/atlas/security-vpc-peering/#in-configure-your-network-peering-connection."
 	VNet                                      = "Name of your Azure VNet."
 	ResourceGroup                             = "Name of your Azure resource group."
 	IAMAssumedRoleARN                         = "Role ARN that Atlas assumes to access your AWS account."
 	DirectoryID                               = "Unique identifier for an Azure AD directory."
 	SubscriptionID                            = "Unique identifier of the Azure subscription in which the VNet resides."
 	GCPProjectID                              = "Unique identifier of the GCP project in which the network peer resides."
-	Network                                   = "Unique identifier of the Network Peering connection in the Atlas project."
+	Network                                   = "Human-readable label in GCP that identifies the network to peer with the Atlas VPS."
```
Comment (kyuan-mongodb) [inline internal/usage/usage.go:272]:
```suggestion
	Network                                   = "Human-readable label in GCP that identifies the network to peer with the Atlas VPC."
```

**Review by kyuan-mongodb — APPROVED** [review]
LGTM % a question and a potential typo.

File: internal/cli/atlas/networking/peering/create/azure.go (hunk 1 above)
Comment (sarahsimpers) [reply internal/cli/atlas/networking/peering/create/azure.go:129]: I think you're right... I took this from the API but I will change it to VPC

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**